### DISCUSSION TITLE
About graden's mark on the mini map

### DISCUSSION THREAD

**Discussion by Z1woO** [Aelto/W3_RandomEncounters_Tweaks#68] [General]
How can I turn off Graden's marker on the mini map?

 I've been waiting for updates for more than ten days:（

**Comment by Aelto** [comment]
i haven't progressed at all as i was not home for many weeks. If this is something you really want, you can download the pre_release 3 which didn't show the markers on the minimap. There were not many changes between the pr3 and pr4 so you can roll back without any issue. I think you can even replace only RER and not the sharedutils and it will work without re merging anything.

**Reply by Z1woO** [reply]
okay, thank you. I see if mod is updated every day, as if I have OCD>.<... Looking forward to the next version update, I wish you all the best:)
